### DISCUSSION TITLE
No permission to edit file

### DISCUSSION THREAD

**Discussion by LAPIII** [AntiMicroX/antimicrox#534] [General] [answered]
Upon saving my first profile, I got this: 

![Save_Config 05-09-2022 12⦂15⦂47⦂817 PM](https://user-images.githubusercontent.com/7947363/188488489-97a24867-560f-4119-bf02-b02c265eccd9.jpg)

I am that administrator and only user of this PC. What should I do?

**Comment by pktiuk** [comment]
I think you should try to delete this file just before pressing save button in AntiMicroX.

**Comment by LAPIII** [comment]
But the file won't exist anywhere until i press the save button in AntiMicroX and then the one in the File Explorer window.

**Comment by LAPIII** [comment, marked as answer]
I just had to run AntiMicroX as administrator.
